app/assets/javascripts/oxalis/view/action_bar_view.js

```diff
@@ -85,6 +85,9 @@ class ActionBarView extends React.PureComponent<Props, State> {
   render() {
     const isTraceMode = this.props.controlMode === ControlModeEnum.TRACE;
     const isVolumeSupported = !Constants.MODES_ARBITRARY.includes(this.props.viewMode);
+    const isArbitrarySupported =
+      (this.props.controlMode === ControlModeEnum.TRACE && this.props.hasSkeleton) ||
```

Comment:
I'd say these two lines are equivalent to `this.props.hasSkeleton || this.props.controlMode === ControlModeEnum.VIEW` as `ControlModeEnum` only contains the two values VIEW and TRACE.